vendor/github.com/fnproject/fn_go/modelsv2/fn.go

```diff
@@ -23,6 +23,9 @@ type Fn struct {
 	// App ID.
 	AppID string `json:"app_id,omitempty"`
 
+	// Architecture type for the application
+	Architecture []string `json:"architecture"`
```

Comment:
Should it be 'architectures' rather than 'architecture'?

Reply:
Done.